docs/DynamicESDL/DynamicESDL_Includer.xml

```diff
@@ -201,9 +192,7 @@
                     fileref="images/OSSgr3.png" /></entry>
 
                     <entry>You can use this command to confirm HPCC processes
-                    are stopped (on Centos/Red Hat):<para> <programlisting>sudo /sbin/service hpcc-init status</programlisting>
-                    <phrase>For Ubuntu</phrase> <programlisting>sudo service hpcc-init status
-</programlisting> <phrase>For Debian 6 (Squeeze)</phrase> <programlisting>sudo /etc/init.d/hpcc-init status</programlisting>
+                    are stopped (on Centos/Red Hat):<para> <programlisting>sudo /etc/init.d/hpcc-init status</programlisting>
```

Comment:
Don't think we need the (on Centos/redhat) comment here.  It will be the same across all distros.